hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -207,6 +207,14 @@ public enum ChecksumCombineMode {
       tags = ConfigTag.CLIENT)
   private long excludeNodesExpiryTime = 10 * 60 * 1000;
 
+  @Config(key = "ec.reconstruct.validation",
+      defaultValue = "false",
+      description = "The flag whether to validate that EC reconstruction tasks " +
+          "reconstruct target containers correctly. When validation fails, " +
+          "reconstruction tasks will fail.",
+      tags = ConfigTag.CLIENT)
+  private boolean ecReconstructValidation = false;
```

Comment:
Are there any drawbacks to having this feature enabled? If we commit it as disabled it will likely never be turned on and we may get burned by a correctness issue this could have caught. If it is relatively safe we may not even want/need a config key to disable it.

Reply:
Thank you for your reviewing! I have updated the description of this PR. In fact, we use the data obtained after decoding for another decoding and compare it with the original data to verify whether the decoding is correct. This may affect the speed of reconstruction. Maybe whether to enable this feature may need to be decided by user.